fitbit-react/src/Util/Toasts/toasts.css

```diff
@@ -0,0 +1,17 @@
+.toasts-container{
+    z-index: 1000;
+    position: fixed;
+    top: 1rem;
+    right: 10px;
+}
+.toast{
+    min-width: 200px;
```

Comment:
Can use rem instead of px, especially for 200px.

Reply:
Done